MapboxVision.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "MapboxVision"
-  s.version      = "0.3.0"
+  s.version      = "0.3.1"
```

Comment:
There's no need to update the version. It's a part of the release process.

Reply:
Got it.